### DISCUSSION TITLE
How is this balanced

### DISCUSSION THREAD

**Discussion by dIKOS00** [ValveSoftware/Dota2-Gameplay#14866] [General]
7526612939 This have any sense some people from Valve do something, all time is the same.

**Comment by cedricchi** [comment]
I think for Valve "balance" means keeping you hooked to the game for eternity by forcing 50% winrate. When you have win streak you will be given the worst possible player worse than your imagination. These bots will feed, troll, jungle to make sure you will lose. 
Valve will never make their matchmaking algorithm public, because of the dark nature of their business strategy. 
On the other hand, people are not as foolish as @jeffhill, they simply leave dota 2 once they see the ridiculous quality of game experience.
The ones that are left use unfair services such as Overplus, boosting, account selling on Taobao, truly toxic people.
Recently they added the feature that allow Dota+ player to deny a low quality match only to be given even worse team. I personally experienced 11-games losing streak!!! I want to vomit 
By the way, the skill differences of our games are so huge you can compare them to heaven and earth. Stomps make me want to kill myself.

**Reply by tiagoacapeleiro** [reply]
when you start to play a game (every type of game) you start learning how to play, and when you don't know how to play you will lose more games than win, as you get better and improve this tendency of losing will change, now you win more than lose, when you reach you skill ceiling you get stuck on 50%, and this is normal. If the game had trash matchmaking you would probably have more 50% win rate or less than 50%. I'm sorry to say but the problem isn't the game it's you, if you want a higher win rate and more MMR you need to improve some aspects of your game that you 100% are lacking, otherwise you would not be stuck at 50%.

**Reply by cedricchi** [reply]
LOL
When you are late for work it is not the traffic that is so obviously jammed everyone can see. The problem is you. If you chose the good route you would reach office on time. If you chose the wrong one then you get caught, as you get better at picking route you will be a good worker.
HAHA, what a joke you boots licking Valve monkey

**Reply by tiagoacapeleiro** [reply]
Hey man, listen. When I first started playing Dota 2, I was stuck at 2k MMR. But over time, I managed to climb all the way to 5k. Everyone improves at their own pace.  If you feel stuck in your MMR bracket, it might be because you haven't quite mastered the skills needed to move up yet. It's tough to hear, but it's true. I have a bunch of friends who've also made similar climbs. If you're looking for inspiration, check out Grubby's Road. He's a former pro Warcraft 3 player who picked up Dota 2 a year or two ago (can't remember exactly). He started out in Guardian and climbed all the way to High Divine.

**Reply by cedricchi** [reply]
Hey men, you listen to me.
I have played Dota since War3. It's not that I can't rank up, it's the system so very very ridiculous. From Herald to Immortal same trolls feeding like brain dead, same support sniper building as carry not buying a single sentry, same jungle bot pos 1 not even looking at lane creeps or take free tower, same skill gap so huge even ocean can't fill, same rigged game after win streak that force you to lose making you vomit or smash screen like Gaben. After all, he wanted this and he deserves this. But we simply want fair and random matches just like Valve did for 8000mmr players. 
Why on earth do you think Valve did this change for 8000 mmr huh? Think before you speak

**Reply by cedricchi** [reply]
BTW, ASK Grubby why he quit Dota2, you will surprise yourself

**Reply by tiagoacapeleiro** [reply]
i know exactly why grubby left dota, because the **chat** was too toxic with him. Please send me a print of your behavior score

**Reply by cedricchi** [reply]
Will do tmr morning 
Grubby left because of match making even he was playing on EU server.

**Reply by tiagoacapeleiro** [reply]
Nops

**Reply by tiagoacapeleiro** [reply]
he literally left because the chat was too toxic while he was playing, that's one of the reason he recently said that if he plays dota it will be sub only, u r just lying to defend your point.

**Reply by cedricchi** [reply]
One thing you should understand is all the problems including toxic community came from matchmaking.
Now here is a video of Grubby talking about matchmaking https://youtu.be/dmWZciCKJS0?si=kiufxulE-XCJz0f7
You are a typical Valve pet, all these talks without a single piece of evidence to support what you are saying.
This is the last evidence I am giving you. I have nothing to defend, "truth is truth whether you like or not". Keep saying whatever you want to say.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/57655363/71d7aef7-4466-4d7f-baf7-e6c87415c1a1)

**Reply by tiagoacapeleiro** [reply]
you could say the matchmaking isnt good, but u cant say its rigged, by the simple fact that if it was, nobody was able to climb mmr, if u give me your account (when i used to play dota) I could climb from your rank to ancient 5 in 3 weeks. with no problem whatsoever.

**Reply by cedricchi** [reply]
I have nothing to say to you. You don't even play this game now. You are muted.

**Reply by tiagoacapeleiro** [reply]
lmao, archon got mad, looks like you are exactly the type of player that are making the matchmaking "RiGgED" hashahahahahahahahah and btw i do play this game, i started play again with this new patch 7.36

**Reply by tiagoacapeleiro** [reply]
> I have nothing to say to you. You don't even play this game now. You are muted.

https://www.youtube.com/watch?v=EN_iUhaaFV0
